lxd/bgp/server.go

```diff
@@ -435,6 +436,13 @@ func (s *Server) addPeer(address net.IP, asn uint32, password string) error {
 			Enabled:     true,
 			MultihopTtl: 255,
 		},
+
+		// Configure holdtime default is cisco like 180s
```

Comment:
This sentence doesn't read well, can you clarify what you mean here please. Thanks
Also comments should end with a full stop.

Reply:
I think we can probably just lose that comment as the default 180s bit isn't handled here anyway.

Reply:
full stop added thanks.
Good or bad but most most of network engineers this of cisco implementation as reference. Here I just try to clarify why i choose this value. As I know Mikrotik is also using 3minutes, but Juniper prefer 90 sec. interval.
I've changes it a little bit, what do you think.

Reply:
The default value seems fine, as you say its used in the industry, it was rather the location of the comment where the default wasn't referenced that concerned me. Its fine now thanks.